README.md

```diff
@@ -29,29 +41,43 @@ Automatic collection of lifecycle-events requires API level 15 and up (Ice Cream
 
 * **[1.0-Alpha.5]** Setup and start of the Application Insights SDK are now done using the new umbrella class `AppInsights` instead of `TelemetryClient`
 
-## Setup ##
-	
-**Add a compile dependency for the SDK**
+##<a name="3"></a> Setup
+
+This is the recommended way to setup Application Insights for your Android app. For other ways to setup the SDK, see [Advanced Setup](#4).
+
+We're assuming you are using Android Studio and gradle to build your Android application.
+
+### 1. **Add a compile dependency for the SDK**
 
-Per-module
+In your module's ```build.gradle```add a dependency for Application Insights 
 
-```java
+```groovy
 dependencies {
-    compile 'com.microsoft.azure:applicationinsights-android:+'
+    compile 'com.microsoft.azure:applicationinsights-android:1.0-beta.3'
 }
 ```
 
-**Configure the instrumentation key and add permissions**
+### 2. **Configure the instrumentation key
 
 Please see the "[Getting an Application Insights Instrumentation Key](https://github.com/Microsoft/ApplicationInsights-Home/wiki#getting-an-application-insights-instrumentation-key)" section of the wiki for more information on acquiring a key.
+
+### 3. Add permissions
 
-Add the two permissions for `INTERNET` and `ACCESS_NETWORK_STATE` into your app's `AndroidManifest.xml` as well as the property for your instrumentation key as follows. Replace `${AI_INSTRUMENTATION_KEY}` with your instrumentation key or the variable leave it and use gradle.properties to set it. 
+Add the two permissions for `INTERNET` and `ACCESS_NETWORK_STATE` to your app's `AndroidManifest.xml`
 
 ```xml
 <manifest>
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    
+</manifest>
+```
+
+### 4. Add your instrumentation key to Manifest
+
+Ad the _instrumentation key_ for your app to your Android Manifest as follows. Replace `${AI_INSTRUMENTATION_KEY}` with your instrumentation key. You can leave the variable as is if you want to use your ```gradle.properties``` to set it (see [Advanced Setup](#4)). 
```

Comment:
THX Chris :)

Von: Christoph Wendt
Antworten an: Microsoft/ApplicationInsights-Android
Datum: Mittwoch, 22. April 2015 16:27
An: Microsoft/ApplicationInsights-Android
Cc: Benjamin Reimold
Betreff: Re: [ApplicationInsights-Android] Updating and rewriting the readme for beta.3 (#23)


In README.md<https://github.com/Microsoft/ApplicationInsights-Android/pull/23#discussion_r28876999>:

>
>  ```xml
>  <manifest>
>      <uses-permission android:name="android.permission.INTERNET" />
>      <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
> -
> +</manifest>
> +```
> +
> +### 4. Add your instrumentation key to Manifest
> +
> +Ad the _instrumentation key_ for your app to your Android Manifest as follows. Replace `${AI_INSTRUMENTATION_KEY}` with your instrumentation key. You can leave the variable as is if you want to use your ```gradle.properties``` to set it (see [Advanced Setup](#4)).


Typo: Add

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/ApplicationInsights-Android/pull/23/files#r28876999>.